.spectral.yaml

```diff
@@ -0,0 +1,3 @@
+extends: ["spectral:oas"]
+rules:
+  operation-description: off
```

Comment:
🤔  Why is it disabled?
from documentation https://meta.stoplight.io/docs/spectral/4dec24461f3af-open-api-rules#operation-description it's recommended, furthermore "**description**" should be always filled in each API specs.